pyscraper/get_links_from_ep.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/python
+
+from everypolitician import EveryPolitician
+
+def output_file(country, legislature, filename):
+    data = EveryPolitician().country(country).legislature(legislature)
+    output_filename = "../members/{0}.xml".format(filename)
+    fp = open(output_filename,"w")
+    fp.write('''<?xml version="1.0" encoding="ISO-8859-1"?>
+    <publicwhip>
+    ''')
+
+    for person in data.popolo().persons:
+        parlparse_id = person.identifier_value('parlparse')
+        if parlparse_id is not None:
+            props = {}
+            if person.twitter is not None:
```

Comment:
I think these could be e.g. `if person.twitter:` - if it's blank, that's not much use either.